script.js

```diff
@@ -0,0 +1,60 @@
+
+     
+    let pic = document.getElementById('cnvs');
+    let ctx= pic.getContext('2d');
+
+    changeCanvas(pic.width, pic.height)
```

Comment:
if you use this function as initialization of a canvas, it would be better to leave canvas element in the index.html without
any height and width properties and store it like a constant in your js file: const DEFAULT_CANVAS_SIZE = 512